frontend/src/lib/k8s/index.ts

```diff
@@ -39,53 +39,40 @@ import ServiceAccount from './serviceAccount';
 import StatefulSet from './statefulSet';
 import StorageClass from './storageClass';
 
-const classList = [
-  ClusterRole,
-  ClusterRoleBinding,
-  ConfigMap,
-  CustomResourceDefinition,
-  CronJob,
-  DaemonSet,
-  Deployment,
-  Endpoints,
-  LimitRange,
-  Lease,
-  ResourceQuota,
-  HPA,
-  PodDisruptionBudget,
-  PriorityClass,
-  Ingress,
-  IngressClass,
-  Job,
-  Namespace,
-  NetworkPolicy,
-  Node,
-  PersistentVolume,
-  PersistentVolumeClaim,
-  Pod,
-  ReplicaSet,
-  Role,
-  RoleBinding,
-  RuntimeClass,
-  Secret,
-  Service,
-  ServiceAccount,
-  StatefulSet,
-  StorageClass,
-];
-
-const resourceClassesDict: {
-  [className: string]: KubeObject;
-} = {};
-
-classList.forEach(cls => {
-  // Ideally this should just be the class name, but until we ensure the class name is consistent
-  // (in what comes to the capitalization), we use this lazy approach.
-  const className: string = cls.className.charAt(0).toUpperCase() + cls.className.slice(1);
-  resourceClassesDict[className] = cls;
-});
```

Comment:
Are you sure these are no longer needed?

Reply:
yeah, it constructs the ResourceClasses object, but now we have the object from the start